judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/contest/ContestStore.java

```diff
@@ -176,6 +212,94 @@ public Contest createContest(ContestCreateData contestCreateData) {
         });
     }
 
+    public String importDump(ContestDump contestDump) {
+        if (contestDump.getJid().isPresent() && contestDao.selectByJid(contestDump.getJid().get()).isPresent()) {
+            throw ContestErrors.jidAlreadyExists(contestDump.getJid().get());
+        }
+        if (contestDao.selectBySlug(contestDump.getSlug()).isPresent()) {
+            throw ContestErrors.slugAlreadyExists(contestDump.getSlug());
+        }
+
+        ContestModel contestModel = new ContestModel();
```

Comment:
Just use `model` and `dump`. For brevity and readability, unless ambiguous (in all places in this PR)

Reply:
Changed